source/is.js

```diff
@@ -3,7 +3,7 @@ import _curry2 from './internal/_curry2.js';
 
 /**
  * See if an object (i.e. `val`) is an instance of the supplied constructor. This
- * function will check up the inheritance chain, if any. 
+ * function will check up the inheritance chain, if any.
```

Comment:
Previous builds were actually failing because of linting errors:
![Screenshot 2021-12-16 at 21 59 48](https://user-images.githubusercontent.com/681975/146462584-bf1ccbaa-6fa9-416e-882a-d1b53bd53dd3.png)